### ISSUE TITLE
Invalid appdata file

### ISSUE THREAD

**Issue by alatiera** [calo001/fondo#41]
Probably cause the last 2 releases use the same date.

```
Running appstream-compose
Processing application com.github.calo001.fondo
com.github.calo001.fondo:          AppData problem: tag-invalid : <release> version was duplicated
Error loading AppData file: AppData file /app/share/appdata/com.github.calo001.fondo.appdata.xml was not valid
Error: ERROR: appstream-compose failed: Child process exited with code 1
```

**Comment by tim77** [comment]
This probably already fixed in recent commits.

**Comment by alatiera** [comment]
Can a new release be made so `appstream-compose` won't complain with the latest one?

**Comment by calo001** [comment]
Fixed in Fondo 1.3.0
